openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -179,6 +181,16 @@ def update_account_settings(requesting_user, update, username=None):
             }
             del update[read_only_field]
 
+    additional_fields = {"sch_org","phone", "organization_type"}.intersection(set(update.keys()))
+    if additional_fields:
+        try:
+            instance = AdditionalRegistrationFields.objects.filter(user=existing_user).first()
+            for field in list(additional_fields):
+                setattr(instance, field, update[field])
+            instance.save()
+        except ObjectDoesNotExist:
+            log.info('Addition fields record does not exist')
```

Comment:
Should we return from the function if the object doesn't exist? and shouldn't we except `AdditionalRegistrationFields.DoesNotExist` exception?

Reply:
we should not return from there in case if records does not exist because multiple fields are being update in this function. Fewer fields are being updated after this code block.